src/components/DefaultSectionContainer/DefaultSectionContainer.test.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { render, screen } from '../../test/testUtils';
+import { DefaultSectionContainer, DefaultSectionContainerProps } from './DefaultSectionContainer';
+
+const defaultProps: DefaultSectionContainerProps = {
+  children: <div>children</div>,
+  title: 'My title',
+};
+
+const setup = (props = {}) => {
+  const combinedProps = { ...defaultProps, ...props };
+  render(<DefaultSectionContainer {...combinedProps} />);
+};
+
+describe('The DefaultSectionContainer component', () => {
+  it('correctly renders the title in kebab-case and sets the id attribute accordingly.', () => {
+    setup();
+
+    expect(screen.getByTestId('container')).toHaveAttribute('id', 'my-title');
```

Comment:
It would be possible to get the container without using a test-id:
```suggestion
    expect(screen.getByText('children').parentElement).toHaveAttribute('id', 'my-title');
```